lsp-mode.el

```diff
@@ -715,6 +715,36 @@ are determined by the index of the element."
                          (const kind)))
   :group 'lsp-imenu)
 
+(defcustom lsp-imenu-index-symbol-kinds '(5 6 7 8 9 10 11 12 23)
+  "Which symbol kinds to show in imenu."
+  :type '(repeat (choice (const :tag  "File" 1)
```

Comment:
Please check `lsp-protocol`, we have enum value for those number now.
Ex: `lsp/symbol-kind-file`.
Please use them instead of number like this